test/unit/smartapp-initialize-spec.js

```diff
@@ -0,0 +1,130 @@
+const assert = require('assert').strict
+const SmartApp = require('../../lib/smart-app')
+
+describe('smartapp-initialize-spec', () => {
+	it('default handler', async () => {
```

Comment:
If we aren't using more specific verbiage in the test names, I like using `test` better instead of `it`.